ctp2_code/gs/gameobj/CityData.cpp

```diff
@@ -6598,8 +6605,7 @@ void CityData::ResetCityOwner(sint32 owner)
 	NewGovernment(g_player[m_owner]->m_government_type);
 
 	m_walls_nullified = false;
-	m_franchiseTurnsRemaining = 0;
-	m_franchise_owner = -1;
+	RemoveFranchise(); // Franchise removed in contrast to conversion, see below
```

Comment:
@MartinGuehmann do you see a problem here effectively replacing `m_franchiseTurnsRemaining = 0;` with `m_franchiseTurnsRemaining = -1;`?

Reply:
Well, I would assume with our current changes it should also be -1. Simply, this resets the city to default values.

But actually what is RemoveFranchise() doing? Does it set m_franchiseTurnsRemaining to -1?

Reply:
Yes, see https://github.com/civctp2/civctp2/pull/166/commits/6283ed8472666e696946dcdaecf460f7a3dfce9e#diff-647de8d8715871ce5f3623bf0a4f0158R5976